### PR TITLE
feat: add mercury summary order

### PR DESCRIPTION
**Pull request by SamuelBenkeCalabresi**
What's changed
* feat: add mercury summary order

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204742659776442
  - https://app.asana.com/0/0/1204754574457049

### CODE REVIEW

File: apps/lineup-mercury/src/app/components/Basket/Basket.tsx [hunk 1]
```diff
@@ -1,27 +1,18 @@
...
-} from '@line-up/line-up-props'
+import { Box, Button, priceFormatterConstructor, Text, EntityId } from '@line-up/line-up-props'
 import { IPerformanceDetailed, useLineUp } from '@line-up/lineup-hooks'
-import { useContext } from 'react'
 import { useIntl } from 'react-intl'
 import { FormLock, Trash } from 'grommet-icons'
 import { BasketItemList } from '../BasketItem'
 
 interface BasketProps {
   performance?: IPerformanceDetailed
```
Comment (nananathanh) [inline apps/lineup-mercury/src/app/components/Basket/Basket.tsx:8]: Is theperformance used in this component?

File: apps/lineup-mercury/src/app/containers/Order/Order.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  return (
+    <Box>
+      <Box direction="row" gap="medium" justify="between">
+        <Box>
+          <Heading level={2}>Time to Get Excited...</Heading>
+          <Text>
+            Your tickets are booked and you're all set. Come back to this page anytime to upgrade
+            your order, access your tickets and to find out more.
+          </Text>
```
Comment (nananathanh) [inline apps/lineup-mercury/src/app/containers/Order/Order.tsx:16]: Should these also be dependent on if the show was in the past?

**Review by nananathanh — COMMENTED** [review]
If I refresh the page after finishing my order the Order complete page doesn't show any Order details.

I'm guessing this is because the basket state is empty once refreshed and we haven't retrieved the transaction from the order.

File: apps/lineup-mercury/src/app/containers/Order/Order.tsx (hunk 2 above)
Comment (SamuelBenkeCalabresi) [reply apps/lineup-mercury/src/app/containers/Order/Order.tsx:16]: @nananathanh this message is not dependent on if the show was in the past at the moment, but I can update it to be dynamic if we want to. In Marvel this doesn't seem to change

**Pull request merged**